region/tourian/main/Metroid Room 2.json

```diff
@@ -14,7 +14,26 @@
       "nodeType": "door",
       "nodeSubType": "blue",
       "nodeAddress": "0x001a9c0",
-      "doorEnvironments": [{"physics": "air"}]
+      "doorEnvironments": [{"physics": "air"}],
+      "locks": [
```

Comment:
Oh interesting. I'm not sure if this will work properly. @blkerby was saying it might make it think the door is locked, even though it says it's blue. We may need another solution, he mentioned making a node 3 by node 1 and it could be applied there.

Reply:
Once obstacle "A" (Metroids) is cleared, you could freely traverse to node 2 and acquire the flag using the lock there. So I'm not sure there's a reason to have another lock at (or near) node 1?